pure-pso/templates/plugin/node-server.yaml

```diff
@@ -43,6 +43,13 @@ spec:
       # https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
       dnsPolicy: ClusterFirstWithHostNet
       hostPID: true
+      initContainers:
```

Comment:
Do we want to add something that makes this optional, in case cluster admins don't want packages installed automagically?

Reply:
Interesting idea. Might be a good idea, but the default is to install.

Reply:
Done